pkg/controllers/nodeclaim/expiration/controller.go

```diff
@@ -78,6 +78,11 @@ func (c *Controller) Reconcile(ctx context.Context, nodeClaim *v1.NodeClaim) (re
 		return reconcile.Result{RequeueAfter: expirationTime.Sub(c.clock.Now())}, nil
 	}
 	// 3. Otherwise, if the NodeClaim is expired we can forcefully expire the nodeclaim (by deleting it)
+	stored := nodeClaim.DeepCopy()
+	nodeClaim.StatusConditions().SetTrueWithReason(v1.ConditionTypeDisruptionReason, v1.DisruptionReasonExpired, "nodeClaim expired")
```

Comment:
Same comment here: Can we add a more descriptive message to this status condition?

Reply:
added more info, please do let me know if you feel we need some additional information